### ISSUE TITLE
Custom lists are random

### ISSUE THREAD

**Issue by mizaki** [alisww/py-manga#8]
Sorry can't do these as pull requests right now.

List stats does not offer a static list. I presume if an item, of what appears to be five, is empty it's not rendered.

series_parser.py
line 447: 
```
    read_lists = contents[13].find_all("b")
    manga["reading_lists"] = {
        "reading": read_lists[0].get_text().strip(),
        "wish": read_lists[1].get_text().strip(),
        "unfinished": read_lists[2].get_text().strip(),
        "custom": read_lists[3].get_text().strip(),
    }
```
replaced with:
```
    manga["reading_lists"] = dict()
    cnt = ""
    for obj in contents[13]:
        if "<b>" in str(obj):
            cnt = obj.get_text().strip()
        if "lists" in str(obj):
            if "reading" in str(obj):
                manga["reading_lists"]["reading"] = cnt
            if "wish" in str(obj):
                manga["reading_lists"]["wish"] = cnt
            if "completed" in str(obj):
                manga["reading_lists"]["completed"] = cnt
            if "unfinished" in str(obj):
                manga["reading_lists"]["unfinished"] = cnt
            if "custom" in str(obj):
                manga["reading_lists"]["custom"] = cnt
```

Probably a better way to do it? Logic loop through the list stats content. Save the number of the list. Discover the list name and used the previously saved number.

**Comment by kore-signet** [comment]
hi! can you re-post the code using a code block (```)? just for ease of reading haha.
thank you for flagging these edge-cases!

**Comment by mizaki** [comment]
I've edited it and it should be good now. Been a while since I've used github :)

**Comment by kore-signet** [comment]
incorporated your fix in 44099f9f6747b107a68ee0f0c569f7f90812d13f ! thank you so much :D
